diff --git a/dist/vue-tippy.d.ts b/dist/vue-tippy.d.ts
index 760b1c8f97d3d548a79a0d35a9bc36a53089822a..0f624e8c5ac52703c5050fc900f37f0f352c049c 100644
--- a/dist/vue-tippy.d.ts
+++ b/dist/vue-tippy.d.ts
@@ -10,6 +10,7 @@ import { Props } from 'tippy.js';
 import { PropType } from 'vue';
 import { Ref } from 'vue';
 import { roundArrow } from 'tippy.js';
+import { ShallowRef } from 'vue';
 import tippy from 'tippy.js';
 import { UnwrapNestedRefs } from 'vue';
 import { VNode } from 'vue';
@@ -722,7 +723,7 @@ export declare function useSingleton(instances: TippyInstances, optionalProps?:
     singleton: import("vue").Ref<import("tippy.js").CreateSingletonInstance<CreateSingletonProps<Props>> | undefined>;
 };
 
-export declare function useTippy(el: TippyElement | (() => TippyElement) | Ref<TippyElement> | Ref<TippyElement | undefined>, opts?: TippyOptions, settings?: {
+export declare function useTippy(el: TippyElement | (() => TippyElement) | Ref<TippyElement> | Ref<TippyElement | undefined> | ShallowRef<TippyElement | null>, opts?: TippyOptions, settings?: {
     mount: boolean;
     appName: string;
 }): {
